README.md

```diff
@@ -64,6 +64,27 @@ Create variables in Airflow (**Admin > Variables** in the UI):
 
 Check other variables in `dags/ethereumetl_airflow/variables.py`.
 
+### Updating package requirements
+Suggested package requirements for Composer are stored in `requirements_airflow.txt`.
+
+You can update the Composer environment using the follow script:
+```
+COMBINED_REQUIREMENTS_PATH="$(mktemp)"
```

Comment:
Nit: "combined" might be confusing here, since we're not combining requirements from different files.